tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -464,6 +466,7 @@ Status IrEmitterUnnested::HandleCustomCall(HloInstruction* custom_call) {
     return Status::OK();
   }
 
+#if !TENSORFLOW_USE_ROCM
```

Comment:
It's unfortunate that the XLA compiler has a CUDA header dependency. It shouldn't be there. Anyway, using `#if` is justified in this case. Can you use something like `#if TENSORFLOW_USE_CUDA` rather than `!TENSORFLOW_USE_ROCM`? That'd be cleaner in my opinion?

Reply:
@thomasjoerg need your guidance here. there is really no such macro as `TENSORFLOW_USE_CUDA` but there is `GOOGLE_CUDA`.

but in our earlier PRs it seems use of `GOOGLE_CUDA` is disallowed within XLA so I resorted to use `!TENSORFLOW_USE_ROCM`.

Reply:
a quick grep within XLA shows only `nccl_all_reduce_thunk.cc` has references to `GOOGLE_CUDA` macro.